src/libDirectoryService/MicroBlockProcessing.cpp

```diff
@@ -406,13 +406,8 @@ bool DirectoryService::ProcessMicroblockSubmissionFromShard(
   }
 #endif  // DM_TEST_DM_LESSMB_ONE
 
-#ifdef DM_TEST_DM_LESSMB_ALL
-  if (m_mediator.m_ds->m_mode == BACKUP_DS) {
-    LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
-              "Letting all of the backups refuse some Microblock submission "
-              "(DM_TEST_DM_LESSMB_ALL)");
-    return false;
-  }
+#if 1
```

Comment:
Any reason to remove this?

Reply:
This is just for tetsing on small scale. Will remove.